doc/api/readline.md

```diff
@@ -527,6 +528,107 @@ rl.on('line', (line) => {
 });
 ```
 
+## TTY keybindings
+
+<table>
+  <tr>
+    <th>Keybindings</th>
+    <th>Description</th>
+  </tr>
+  <tr>
+    <td><code>ctrl+shift+backspace</code></td>
+    <td>delete line left</td>
+  </tr>
+  <tr>
+    <td><code>ctrl+shift+delete</code></td>
+    <td>delete line right</td>
+  </tr>
+  <tr>
+    <td><code>ctrl+c</code></td>
+    <td>emits SIGINT</td>
```

Comment:
`emits` -> `emit` for consistency with other cells?

Reply:
done